src/app/tsweekly.service.ts

```diff
@@ -57,17 +60,43 @@ export class TsweeklyService {
     }
 
     getWeeklies(email: string, weekid: number): Observable<TsWeekly> {
+
         console.log('Getting weeklies for', email, weekid);
-        return this.http.get<TsWeekly[]>(this.configUrl + '/' + email + '/' + weekid).pipe(
-            mergeMap((items: TsWeekly[]) => from(items)),
+
+        const token = 'Bearer ' + this.oAuthService.getIdToken();
```

Comment:
I think you can use `localStorage.getItem` as you saved the token in there on login (`app.component.ts`) - same either way I guess